src/main/java/org/cryptacular/bean/EncodingHashBean.java

```diff
@@ -64,9 +71,44 @@ public void setCodecSpec(final Spec<Codec> codecSpec)
   }
 
 
+  /**
+   * Whether data provided to {@link #hash(Object...)} includes a salt as the
+   * last parameter.
+   *
+   * @return  whether hash data includes a salt
+   */
+  public boolean isSalted()
+  {
+    return salted;
+  }
+
+
+  /**
+   * Sets whether {@link #hash(Object...)} should expect a salt as the last
+   * parameter.
+   *
+   * @param  salted  whether hash data includes a salt
+   */
+  public void setSalted(final boolean salted)
+  {
+    this.salted = salted;
+  }
+
+
   @Override
```

Comment:
Sure. I think it's worth noting that there are only two implementations of HashBean. So saying this is the only hash method that works this way is true, but there is also only one that doesn't work this way.